modules/terraform/Makefile

```diff
@@ -34,6 +34,12 @@ terraform/validate:
 terraform/tflint:
 	tflint --enable-plugin=aws
 
+# Auto-fix tflint errors
+terraform/tflint-fix:
+	@for v in $$(find . -type f -name '*.tf'); do \
+		sed -i 's/\([[:alpha:]]\+[0-9]*\)\.\([*0-9]\)\./\1[\2]\./g' "$$v" ; \
+	done
```

Comment:
Let's call this recipe something more specific, perhaps `tflint-fix-deprecated-index` and then have `tflint-fix` depend on it, both do document what this recipe fixes and to allow for future additions.

Also, I think the `sed` command should be

```
s/\([[:alpha:]]\+\)\.\([*0-9]\+\)\./\1[\2]\./g
```

but I don't think in practice it will matter.